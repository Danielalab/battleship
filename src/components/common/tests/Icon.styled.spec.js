import 'jest-styled-components';
import { render } from '@testing-library/react';
import Icon from '../Icon.styled';

describe('Icon', () => {
  test('Should renders Icon component', () => {
    const { container } = render(<Icon />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
