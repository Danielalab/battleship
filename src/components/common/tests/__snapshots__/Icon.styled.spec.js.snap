// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Should renders Icon component 1`] = `
.c0 {
  padding: 4px;
  visibility: visible;
  -webkit-transition: all 0.05s ease-in;
  transition: all 0.05s ease-in;
}

<div>
  <img
    class="c0"
  />
</div>
`;
